alicloud/table_alicloud_vpc_route_table.go

```diff
@@ -0,0 +1,112 @@
+			{Name: "resource_group_id", Type: proto.ColumnType_STRING, Description: "The ID of the resource group to which the VPC belongs."},
+			{Name: "route_entrys", Type: proto.ColumnType_JSON, Hydrate: getVpcRouteTableEntryList, Transform: transform.FromField("RouteEntrys.RouteEntry"), Description: "Route entry represents a route item of one VPC route table."},
+			{Name: "owner_id", Type: proto.ColumnType_STRING, Description: "The ID of the owner of the VPC."},
+			// Other columns
+			{Name: "tags", Type: proto.ColumnType_JSON, Transform: transform.FromField("Tags.Tag"), Description: resourceInterfaceDescription("tags")},
+			{Name: "title", Type: proto.ColumnType_STRING, Transform: transform.FromField("RouteTableName"), Description: resourceInterfaceDescription("title")},
+		},
+	}
+}
+
+//// BUILD HYDRATE INPUT
+
+func RouteTableIDFromRouteTable(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	quals := d.KeyColumnQuals
+	id := quals["id"].GetStringValue()
+	item := &vpc.RouteTable{
+		RouteTableId: id,
+	}
+	return item, nil
+}
+
+func listVpcRouteTable(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	client, err := connectVpc(ctx)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_vpc_route_table.listVpcRouteTable", "connection_error", err)
+		return nil, err
+	}
+	request := vpc.CreateDescribeRouteTableListRequest()
+	request.Scheme = "https"
+	request.PageSize = requests.NewInteger(50)
+	request.PageNumber = requests.NewInteger(1)
+
+	count := 0
+	for {
+		response, err := client.DescribeRouteTableList(request)
+		if err != nil {
+			plugin.Logger(ctx).Error("alicloud_vpc_route_table.listVpcRouteTable", "query_error", err, "request", request)
+			return nil, err
+		}
+		for _, i := range response.RouterTableList.RouterTableListType {
+			plugin.Logger(ctx).Warn("alicloud_vpc_route_table.listVpcRouteTable", "tags", i.Tags, "item", i)
+			d.StreamListItem(ctx, i)
+			count++
+		}
+		if count >= response.TotalCount {
+			break
+		}
+		request.PageNumber = requests.NewInteger(response.PageNumber + 1)
+	}
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+func getVpcRouteTableEntryList(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	client, err := connectVpc(ctx)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_vpc_route_table.getVpcRouteTableEntryList", "connection_error", err)
+		return nil, err
+	}
+	request := vpc.CreateDescribeRouteEntryListRequest()
+	request.Scheme = "https"
+	i := h.Item.(vpc.RouterTableListType)
+	request.RouteTableId = i.RouteTableId
+	response, err := client.DescribeRouteEntryList(request)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_vpc_route_table.getVpcRouteTableEntryList", "query_error", err, "request", request)
+		return nil, err
+	}
+	return response, nil
+}
```

Comment:
For `Get` call use `DescribeRouteTableList` API with `RouteTableId` filter.
`DescribeRouteEntryList` API lists details about all `route` entries.
We can add another hydrate function `DescribeRouteEntryList` to store route details.